src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -73,13 +73,20 @@ public async Task<bool> ApplyLaunchVariablesAsync(IDictionary<string, string> en
             }
 
             return false;
+}
+
+        // TODO: remove when Web Tools is no longer calling this method.
+        public Task StartSessionAsync(CancellationToken cancellationToken)
+        {
+            return StartSessionAsync(runningUnderDebugger: false, cancellationToken);
         }
 
-        public async Task StartSessionAsync(CancellationToken cancellationToken)
+        public async Task StartSessionAsync(bool runningUnderDebugger, CancellationToken cancellationToken)
         {
             if (!_sessionActive)
```

Comment:
Is there a reason this no-ops if called when already started? What if I try to start with `runningUnderDebugger` false the first time, then true the second time? Unless there's a compelling reason not to, consider throwing an exception here if already active.